templates/list_pages/node--oe-list-page--full.html.twig

```diff
@@ -24,15 +25,11 @@
     <div class="row">
       <div class="col-md-6 col-lg-8 align-self-center">
         <h4 class="mb-0">
-          <span class="text-capitalize">{{ label|render|striptags|trim }}</span>
-          ({{ content['extra_field_oe_list_page_total_count:node:oe_list_page']|render|trim }})
+          <span class="text-capitalize">{{ label }}</span>
+          ({{ content['extra_field_oe_list_page_total_count:node:oe_list_page'] }})
         </h4>
       </div>
-      <div class="col-md-12 order-md-2 mt-3">
-        {{ content['extra_field_oe_list_page_selected_filters:node:oe_list_page'] }}
-      </div>
-      <div class="col-md-6 col-lg-4 mt-3 mt-md-0">
-      </div>
+      {{ content['extra_field_oe_list_page_selected_filters:node:oe_list_page'] }}
     </div>
     {{ content['extra_field_oe_list_page_results:node:oe_list_page'] }}
   </div>
```

Comment:
I see some discrepancy to netlify.
But this is likely due to the new bcl version.
What should we do first? Merge the bcl upgrade, or merge list pages?

Reply:
list pages